requirements.txt

```diff
@@ -3,4 +3,6 @@ praw
 pyopenssl
 ndg-httpsclient
 pyasn1
-urllib3
\ No newline at end of file
+urllib3
+simplejson
+requests
```

Comment:
it still used, see reddit.py L14:
"# bug on me PC, see https://urllib3.readthedocs.org/en/latest/security.html section OpenSSL / PyOpenSSL"